conda/cli/common.py

```diff
@@ -281,3 +285,51 @@ def validate_prefix_is_writable(prefix: str) -> str:
     if isdir(dirname(test_path)) and file_path_is_writable(test_path):
         return prefix
     raise EnvironmentNotWritableError(prefix)
```

Comment:
These two functions are really suitable for a consolidation so we have a signature like:

```python
def validate_prefix(prefix, check_writable: bool = False) -> str:
   ...
```

Reply:
I'll take this in a separate PR